Code/src/main/java/nl/utwente/viskell/model/ConstantBox.java

```diff
@@ -0,0 +1,43 @@
+package nl.utwente.viskell.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import nl.utwente.viskell.haskell.expr.Binder;
+import nl.utwente.viskell.haskell.expr.Expression;
+import nl.utwente.viskell.haskell.type.Type;
+
+public class ConstantBox implements Box {
+    
+    private Grouping parent;
+    
+    private Expression expr;
+    
+    private Type type;
+    
+    private OutputPort ouput;
+
+    public ConstantBox(Component parent, String name, Expression expr, Type type) {
```

Comment:
Would changing the value (expression) entail building and wiring up a new ConstantBox, or would `expr` be mutable? Having it mutable would perhaps help SliderBlock, but I'm not sure of the tradeoffs.